canopy/app/saplings/profile.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2019 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+window.$CANOPY.registerApp(function(domNode) {
```

Comment:
Okay for now, but we should establish how we're going to use CanopyJS in these basic saplings.

Reply:
I'm intending to make this a React app sapling and import canopyjs in the actual implementation